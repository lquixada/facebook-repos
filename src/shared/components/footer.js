import React from 'react';
import styled from 'styled-components';

export const Footer = () => (
  <Wrapper>
    <Link href="https://facebook.com/lquixada/">Facebook</Link>
    {' '}|{' '}
    <Link href="https://github.com/lquixada/">Github</Link>
    {' '}|{' '}
    <Link href="https://twitter.com/lquixada/">Twitter</Link>
    <br/>
    © Copyright 2016 Leonardo Quixadá. All rights reserved.
  </Wrapper>
);

const Wrapper = styled.footer`
  margin: 1rem 0;
  padding: 0 1rem;
  color: #767676;
  opacity: .9;
  font-size: 1rem;
  line-height: 1.5;
`;

const Link = styled.a`
  margin-bottom: 1rem;
  color: #767676;
  font-size: 1rem;
  text-decoration: none;
`;
