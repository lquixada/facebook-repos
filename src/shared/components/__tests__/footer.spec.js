import React from 'react';
import {mount} from 'enzyme';

import {Footer} from '../footer';

describe('<Footer />', () => {
  it('matches the snapshot', () => {
    const component = mount(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
