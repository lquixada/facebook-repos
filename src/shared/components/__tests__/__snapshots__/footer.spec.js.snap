// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> matches the snapshot 1`] = `
<Footer>
  <styled.footer>
    <footer
      className="sc-bdVaJa kXnLAJ"
    >
      <styled.a
        href="https://facebook.com/lquixada/"
      >
        <a
          className="sc-bwzfXH LlvFc"
          href="https://facebook.com/lquixada/"
        >
          Facebook
        </a>
      </styled.a>
       
      |
       
      <styled.a
        href="https://github.com/lquixada/"
      >
        <a
          className="sc-bwzfXH LlvFc"
          href="https://github.com/lquixada/"
        >
          Github
        </a>
      </styled.a>
       
      |
       
      <styled.a
        href="https://twitter.com/lquixada/"
      >
        <a
          className="sc-bwzfXH LlvFc"
          href="https://twitter.com/lquixada/"
        >
          Twitter
        </a>
      </styled.a>
      <br />
      Made with ❤ by Leonardo Quixadá.
    </footer>
  </styled.footer>
</Footer>
`;
